text/0107-construct-library-module-lifecycle.md

```diff
@@ -202,14 +202,23 @@ The module is generally available with a backwards compatible guarantee across m
 
 - **Nominate champion** - A member of the Construct Library Squad is assigned as the graduation champion.
 
-- **Clear awslint exclusions** - Review `awslint` exclusions and make the necessary code changes to remove what we can.
-
 - **Create Graduation Issue** - Create a GitHub issue titled `Graduate @aws-cdk/{module} to stable`.
 This issue communicates our intent to graduate this module. It will also serve as an asynchronous channel for anyone who wants to provide feedback.
 Update the module tracking issue with a link to this issue so that the community can be notified.
 
   > External Stakeholders and major community contributors should be contacted directly.
 
+  Include a checklist of tasks that must be completed before flipping the switch.
+  Initially, this would be:
+
+  - **Validate existing bugs** - Make sure there were no *p0* bugs in the last 4 weeks.
+  - **Validate L2 coverage** - Consult module owners and make sure we have sufficient L2 coverage to provide a holistic developer experience,
+  i.e most major use cases can be configured via CDK, without needing prior manual setup.
```

Comment:
Drop the 'prior manual step' addendum or mention it as an example.

This is just one example of where a use case is not completely covered.